docs/staticfiles.md

```diff
@@ -9,6 +9,7 @@ Signature: `StaticFiles(directory=None, packages=None, check_dir=True)`
 * `packages` - A list of strings or list of tuples of strings of python packages.
 * `html` - Run in HTML mode. Automatically loads `index.html` for directories if such file exist.
 * `check_dir` - Ensure that the directory exists upon instantiation. Defaults to `True`.
+* `follow_symlink` - A boolean indicating if symbolic links for files and directories should be followed. Defaults to `False`.
```

Comment:
It's missing on the signature above as well, jfyk

Reply:
thanks, added it.